app/models/room.rb

```diff
@@ -1,3 +1,14 @@
 class Room < ApplicationRecord
   belongs_to :hotel
+  has_many :guest_rooms
+  has_many :guests, through: :guest_rooms
+
+  def add_guest_to_room(guest_id)
+    guest = Guest.find(guest_id)
```

Comment:
This looks great! It's also okay to include these operations in the `create` action of a controller if you'd like, but there's no harm in encapsulating them if it's more readable for you